File: Pb-1/pb-1.py
from datetime import datetime
start_time = datetime.now()
def multiplesof3and5(n):
  sum=0
  for i in range(n):
    if i%3==0 or i%5==0:
      sum+=i
  return sum
        
        
if __name__== "__main__":
  
  print(multiplesof3and5(10))
end_time = datetime.now()
print('Duration: {}'.format(end_time - start_time))

File: Pb-01 Multiples of 3 and 5/pb-1.go
package main
import "fmt"

func MultiplesOf3and5(num int) int{
  sum := 0
  for i := 0; i < num; i++{
    if i % 3 == 0 || i % 5 == 0{
      sum += i
    }
  }
  return sum
}

func main(){
  fmt.Println(MultiplesOf3and5(1000))
}
